internal/pkg/runtime/launch/launcher_linux.go

```diff
@@ -1084,6 +1088,7 @@ func (l *Launcher) setCgroups(instanceName string) error {
 		useCG = true
 	}
 
+	sylog.Debugf("During setting cgroups configuration, whether use cgroup manager: %t", useCG)
```

Comment:
Rather than always including this statement, put a debug statement inside of `useCG` of "Using cgroups manager".  The absence of the message means it will be false.

Reply:
thanks, updated!